libs/partners/nvidia-trt/tests/unit_tests/test_llms.py

```diff
@@ -1,7 +1,35 @@
 """Test TritonTensorRT Chat API wrapper."""
+import sys
+from io import StringIO
+
+import pytest
+from tritonclient.utils import InferenceServerException
+
 from langchain_nvidia_trt import TritonTensorRTLLM
 
 
 def test_initialization() -> None:
     """Test integration initialization."""
     TritonTensorRTLLM(model_name="ensemble", server_url="http://localhost:8001")
+
+
+def test_default_verbose() -> None:
+    llm = TritonTensorRTLLM(server_url="http://localhost:8001", model_name="ensemble")
+    captured = StringIO()
+    sys.stdout = captured
+    with pytest.raises(InferenceServerException):
+        llm.client.is_server_live()
```

Comment:
this is not perfect since it tries to request this address anyway, it might cause cloud/CI/security/etc issues. I don't know. Open for any other ideas.

Reply:
improved here in the recent push